remixer_example/src/main/java/com/google/android/apps/remixer/MainActivity.java

```diff
@@ -49,6 +51,9 @@ protected void onCreate(Bundle savedInstanceState) {
     RemixerFragment remixerFragment = RemixerFragment.newInstance();
     remixerFragment.attachToGesture(this, Direction.UP, 3);
     remixerFragment.attachToButton(this, remixerButton);
+
+    SensorManager sensorManager = (SensorManager)getSystemService(SENSOR_SERVICE);
```

Comment:
You could move this line to the attachToShake method, by calling `activity.getSystemService(...)` and truly make this a one-liner.

Nit: space after the cast.